Detectors/MUON/MID/Workflow/README.md

```diff
@@ -117,6 +117,32 @@ o2-mid-reco-workflow --change-local-to-BC <value>
 
 where `<value>` is the chosen offset in number of BCs (can be negative).
 
+## MID BC filtering
+
+The MID time resolution is better than 25 ns, allowing it to distinguish between different BCs.
+However, a spread of the signal was observed, probably due an insufficient equalization of the delays across the various detection element.
+In order to study and possibly correct for any inefficiency arising from this spread, it is possible to select the BCs corresponding to a collision and merge the digits in a configurable window around this BC.
+To this aim, it is enough to run the reconstruction with the option:
+
+```bash
+o2-mid-reco-workflow --filter-BC <value>
```

Comment:
actually the filter-BC being a bool I think you can’t specify a value, can you ? 
also, seems most of the boolean options we have in O2 follow the naming convention `--enable-xxx` or `disable-xxx`, so I would follow it here, e.g. `--enable-filter-bc` (as the default is false).